### PR TITLE
Create BNFabb233.xml

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
New record.

### CODE REVIEW

File: ParisBNF/abb/BNFabb233.xml [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+                                <locus from="2r"/>
+                                </layout>
+                             </layoutDesc>
+                        </objectDesc>
+
+                        <handDesc>
+                           <handNote xml:id="h1" script="Ethiopic">
+                              <seg type="script">19th-century script.</seg>
+                              <seg type="ink">Black.</seg>
+                               <desc>Crude; according to Antoine d'Abbadie, written by a <persName ref="ETH1274Betaes">falāšā</persName> scribe.</desc>
```
Comment (CarstenHoffmannMarburg) [inline ParisBNF/abb/BNFabb233.xml:74]: As far as I know, the term 'Falasha' is outdated. Being a quotation by Abbadie, it should have quotation marks.

**Review by CarstenHoffmannMarburg — CHANGES_REQUESTED** [review]

File: ParisBNF/abb/BNFabb233.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb233.xml:74]: OK, done.

File: ParisBNF/abb/BNFabb233.xml [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+                </particDesc>
+            <textClass>
+                <keywords>
+                    <term key="Bible"/>
+                    <term key="OldTestament"/>
+                    <term key="Religion"/>
+                    <term key="ChristianLiterature"/>
+                    <term key="BetaEsraelLiterature"/>
+                    <term key="Commentary"/>
+                    <term key="ZaMa"/>
```
Comment (thea-m) [inline ParisBNF/abb/BNFabb233.xml:161]: AmharicLiterature?

File: ParisBNF/abb/BNFabb233.xml [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+                        <collection>Manuscrits orientaux</collection>
+                        <collection>Fonds éthiopien</collection>
+                        <collection>d'Abbadie</collection>
+                        <idno>BnF Éthiopien d'Abbadie 233</idno>
+                  </msIdentifier>
+
+                    <msContents>
+                      <msItem xml:id="ms_i1">
+                          <locus from="1r" to="4v"/>
+                          <title type="incomplete" ref="LIT5982CommOrit" xml:lang="gez"/>
```
Comment (thea-m) [inline ParisBNF/abb/BNFabb233.xml:38]: xml:lang="gez" is not necessary here

File: ParisBNF/abb/BNFabb233.xml [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+                          
+                          <msItem xml:id="ms_i1.1">
+                              <locus from="1r" to="4v"/>
+                              <title type="incomplete" ref="LIT5982CommOrit#Genesis" xml:lang="gez">የኦሪት፡ የፍጥረት፡ ትርጓሜ፡</title>
```
Comment (thea-m) [inline ParisBNF/abb/BNFabb233.xml:42]: should it maybe be xml:lang="am"?
The element textLang could also be added

**Review by thea-m — CHANGES_REQUESTED** [review]
Thank you!

File: ParisBNF/abb/BNFabb233.xml (hunk 2 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb233.xml:161]: ok

File: ParisBNF/abb/BNFabb233.xml (hunk 4 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb233.xml:42]: yes

File: ParisBNF/abb/BNFabb233.xml (hunk 3 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb233.xml:38]: ok

**Review by thea-m — APPROVED** [review]

**Pull request merged**